wa/workloads/pcmark/__init__.py

```diff
@@ -47,6 +47,9 @@ class PcMark(ApkUiautoWorkload):
     def __init__(self, target, **kwargs):
         super(PcMark, self).__init__(target, **kwargs)
         self.gui.timeout = 1500
+        # Do not delete Work 2.0 data-set before each run
+        self.apk.clear_data_on_reset = False
```

Comment:
`self.apk.clear_data_on_reset` shouldn't need to be set directly. `self.clear_data_on_reset` should propagate through to the `PackageHandler`.

Reply:
I added a patch to handle `apk.clear_data_on_reset` within `ApkWorkload`, as it wasn't working out of the box. `PackageHandler` being constructed within `ApkWorkload`'s `__init()__.

Reply:
Apologies I should have said if you set the `clear_data_on_reset` property of the workload itself (for example in the `gfxbench` workload https://github.com/ARM-software/workload-automation/blob/master/wa/workloads/gfxbench/__init__.py#L25) then when initialising the `PackageHandler` the updated value should be propagated without needing to be manually set.

Reply:
Woops, embarrassing mistake...